README.md

```diff
@@ -1,2 +1,56 @@
-# ReactiveXComponent.js
-Javascript API for XComponent
+## Reactive XComponent API
+Reactive XComponent is a javaScript client API that allows you to interact with microservices generated with XComponent software.
+
+## Installation
```

Comment:
Build is better than installation
(because we need to package our api before adding an install paragraph)